addons/web_editor/static/src/js/wysiwyg/widgets/link.js

```diff
@@ -234,10 +234,10 @@ const Link = Widget.extend({
      * @private
      */
     _correctLink: function (url) {
-        if (url.indexOf('mailto:') === 0 || url.indexOf('tel:') === 0) {
-            url = url.replace(/^tel:([0-9]+)$/, 'tel://$1');
-        } else if (url.indexOf('@') !== -1 && url.indexOf(':') === -1) {
+        if (url.indexOf('@') >= 0 && url.indexOf('mailto:') === -1 && !url.match(/^http[s]?/i)) {
```

Comment:
```suggestion
        if (url.includes('@') && !url.includes('mailto:')  && !url.match(/^http[s]?/i)) {
```
Why not use includes? Feels easier to read